configuration/monitor-acs.adoc

```diff
@@ -0,0 +1,14 @@
+[id="monitor-acs"]
+= Monitoring with Prometheus
+include::modules/common-attributes.adoc[]
+:context: monitor-acs
+
+toc::[]
+
+[role="_abstract"]
+link:https://prometheus.io/[Prometheus] is an open-source monitoring and alerting platform.
+You can use it to monitor health and availability of Central and Sensor components of {product-title}.
+
+include::modules/enable-monitoring.adoc[leveloffset=+1]
+
+include::modules/enable-monitoring-customize-port.adoc[leveloffset=+2]
```

Comment:
Just checking whether customizing the port really needed to be indented under the enabling monitoring module?`

Reply:
I think its part of enabling monitoring process for step two - Modify network policies to allow ingress. They can customize the port number and then use the new port number in step 2.